code/index.html

```diff
@@ -1,30 +1,324 @@
+          <br><br>You must select either the "Email only" or "Email and Telemarketing" option.
+          <br><br>You must also select a minimum of <strong>1</strong> option from the Categories list.
+          <br><br>Last of all, you must agree to the terms and conditions that this person is total garbage.
+        </p>
+      </div>
+
+      <button class="subtitle faq__question" id="question4">
+        Is this legal?
+      </button>
+      <div class="faq__answer">
+        <p class="text faq__text">
+          Let us assure you: this is <strong>completely legal</strong>.
+          <br><br>However, this is also <strong>undeniably unethical</strong> and <strong>utterly irreversible</strong>.
+        </p>
       </div>
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+      <button class="subtitle faq__question" id="question5">
+        Help! I'm a victim...
+      </button>
+      <div class="faq__answer">
+        <p class="text faq__text">
+          Our deepest apologies for your misfortune. 
+          <br>Regrettably, this offensive incursion is both anonymous and irreversible.
+          <br><br>We do have some advice that may be useful:
+          <br>• Manually unsubscribe from the newsletters.
+          <br>• Create a new email.
+          <br>• Change your phone number.
+          <br>• Be nicer to people.
+          <br>• Find some new "friends".
+          <br>• Take some time off to reflect on why this might have happened to you.
+          <br><br><em>"Before I was afflicted I went astray: but now have I kept my word."</em> <strong>Psalm 119:67</strong>
       </div>
-    </section>
-    <!-- dont forget to link the javascript file here! -->
-  </body>
+
+    </div>
+
+    <!-- Signup Form Container -->
+    <div class="block-container signup" id="signup">
+      <!-- Form Header -->
+      <img class="signup__image" src="images/death-5.png" alt="Death Note" />
+      <h1 class="title signup__title">
+        R.I.P. INBOX
+      </h1>
+      <!-- Form -->
+      <form id="form" class="signup__form" action="http://httpbin.org/anything" method="post">
+
+        <!-- Name, Email, Phone -->
+        <label for="name">
+          <input name="name" id="name" type="text" placeholder="Their name" required />
+        </label>
+        <label for="email">
+          <input name="email" id="email" type="email" placeholder="Their email" required />
+        </label>
+        <label for="phone">
+          <input name="phone" id="phone" type="tel" placeholder="Their phone number (Optional)" />
+        </label>
+
+        <!-- Radio Buttons -->
+        <!-- Email Only Button -->
+        <div class="select-container">
+          <div class="select-container__object">
+            <label for="emailOnly" class="radio--remove">
```

Comment:
I can see you name your classes with double dash -- , hmmm I wonder if its for a special reason which is in favor of the code...?

Reply:
I was trying to use BEM naming conventions here! They suggested it somewhere in the week's material but I can't find where now - luckily I bookmarked it: http://getbem.com/introduction/